src/main/java/calculator/domain/Calculator.java

```diff
@@ -0,0 +1,48 @@
+package calculator.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Calculator {
+
+    private List<String> formula;
+
+    private final Operator process = new Operator();
+
+    public Calculator(String[] formula) {
+        this.formula = new ArrayList<>(Arrays.asList(formula));
+    }
+
+    public void hasPriority() {
+        for (int i = 0; i < formula.size(); i++) {
+            if (formula.get(i).equals("*") || formula.get(i).equals("/")) {
+                replace(i,
+                    process.intCalculator(formula.get(i - 1), formula.get(i + 1), formula.get(i)));
```

Comment:
```suggestion
                replace(i, process.intCalculator(formula.get(i - 1), formula.get(i + 1), formula.get(i)));
```

이런식으로 작성하시는것이 더 가독성이 좋아보입니다 :)